way_to_home/utils/vehicle_data.py

```diff
@@ -0,0 +1,32 @@
+"""
+    This module use for parsing data got in real time from 'Easy Way' source
+    in GTFS format.
+"""
+
+import requests
+from google import protobuf
+from google.transit import gtfs_realtime_pb2
+
+
+FEED = gtfs_realtime_pb2.FeedMessage()
+RESPONSE = requests.get('http://track.ua-gis.com/gtfs/lviv/vehicle_position')
```

Comment:
is it okay to make the request during reading the file?

Reply:
I fixed it